src/js/requestHandlers.js

```diff
@@ -192,14 +192,25 @@ fluid.defaults("sjrk.storyTelling.server.deleteStoryHandler", {
     invokers: {
         handleRequest: {
             funcName: "sjrk.storyTelling.server.handleDeleteStory",
-            args: ["{arguments}.0", "{server}.deleteStoryDataSource", "{server}.storyDataSource", "{server}.options.globalConfig"]
+            args: ["{that}", "{arguments}.0"]
```

Comment:
"{arguments}.0" should point to "{that}" as well. Passing only one of them should be enough.